x/axelarcork/types/proposal.go

```diff
@@ -0,0 +1,240 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	ProposalTypeAddManagedCellarIDs = "AddAxelarManagedCellarIDs"
```

Comment:
Looks like `gofmt` may not have been run here recently?

Reply:
ran now